src/providers/mbed_provider/utils.rs

```diff
@@ -219,3 +222,79 @@ pub fn psa_export_public_key_size(key_attrs: &psa_key_attributes_t) -> Result<us
         _ => Err(ResponseStatus::PsaErrorInvalidArgument),
     }
 }
+
+/// Wrapper around raw `psa_key_handle_t` which allows for easier manipulation of
+/// handles and the attributes associated with them.
+pub struct Key<'a>(psa_key_handle_t, &'a Mutex<()>);
```

Comment:
Maybe it should be `KeyHandle`?

Reply:
That's how it was named initially, but I renamed it because I thought an operation called `get_key_attributes` sounds a bit weird for a key handle.

Reply:
Fair enough!